src/components/ProgressBars/LinearProgressBar/__stories__/linearProgressBar.stories.mdx

```diff
@@ -0,0 +1,214 @@
+import LinearProgressBar from "../LinearProgressBar";
+import { ArgsTable, Story, Canvas, Meta } from "@storybook/addon-docs";
+import { createComponentTemplate } from "../../../../storybook/functions/create-component-story";
+import colors from "../../../../constants/colors.json";
+import DialogContentContainer from "../../../DialogContentContainer/DialogContentContainer";
+import { Info } from "../../../Icon/Icons";
+import Icon from "../../../Icon/Icon.jsx";
+import { SPINNER, STEPPER, SKELETON } from "../../../../storybook/components/related-components/component-description-map";
+import { Link } from "../../../../storybook/components";
+import BreadcrumbsBar from "../../../BreadcrumbsBar/BreadcrumbsBar.jsx";
+import BreadcrumbItem from "../../../BreadcrumbsBar/BreadcrumbItem/BreadcrumbItem.jsx";
+import "./linearProgressBar.stories.scss";
+
+<Meta
+  title="Components/LinearProgressBar"
+  component={ LinearProgressBar }
+/>
+
+<!--- Component template -->
+
+export const lineProgressBarTemplate = (args) => {
+  return (
+    <div className="linear-progress-bar_small-wrapper">
```

Comment:
please set the progress bar width by using the className prop instead of by using a wrapper